ios-quality-assurance.md

```diff
@@ -1,139 +1,147 @@
-# PROJECT QUALITY ASSURANCE #
+# iOS/Swift project Quality Assurance
 
-The idea behind this document is to have a reference of how to create, develop and maintain an iOS project in Karumi.
+The idea behind this document is to have a reference of how to create, develop and maintain an iOS project in [Karumi](https://karumi.com).
 
-These are no hard rules, but we encourage them as we wrote this document based on our experience doing consultancy projects and in-house project.
+These are no hard rules, but we encourage them as we wrote this document based on our experience doing consultancy projects and in-house project. This document complements the [Project Quality Assurance](https://github.com/Karumi/project-quality-assurance) with all the specifics we follow for Android projects written in Kotlin.
```

Comment:
`Android projects` ? This is the ios-quality-assurance doc.

Reply:
Fixed! I've removed this and another reference to Android in this document.